### PR TITLE
topotato: Migrate test-bgp-community-alias

### PR DESCRIPTION
**Pull request by MrTanoshii**

### CODE REVIEW

File: tests/topotato/bgp-test_community_alias.py [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+
+    @topotatofunc
+    def _bgp_show_prefixes_by_large_community_list(self, topo, r1, r2):
+        expected = {"routes": {str(r2.lo_ip4[0]): [{"valid": True}]}}
+
+        yield from AssertVtysh.make(
+            r1,
+            "bgpd",
+            f"show bgp ipv4 unicast large-community-list r2 json",# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2022 Tsen Chee Vincent LEUNG YIN KO
```
Comment (eqvinox) [inline tests/topotato/bgp-test_community_alias.py:185]: Did something go wrong in rebasing here? The file seems to be duplicated :laughing:

Comment (MrTanoshii) [reply tests/topotato/bgp-test_community_alias.py]: Good catch, my bad

**Comment by MrTanoshii** [comment]
Requires #67

**Comment by eqvinox** [comment]
This is duplicate with #62 :disappointed:

**Comment by eqvinox** [comment]
outdated, closing

**Pull request closed without merging**